x-pack/heartbeat/seccomp_linux.go

```diff
@@ -61,23 +117,16 @@ func init() {
 			"sendto",
 			"set_robust_list",
 			"set_tid_address",
-			"si_code",
 			"sigaltstack",
-			"SIGINT",
```

Comment:
Looking at comment in line 19 I assume there was never a good reason that these were here except copy/paste from somewhere. We likely should do the same for other beats ...

Reply:
This actually broke the syscall filters. Will update the main issue description in a bit.